src/Roslyn/Conventional.Roslyn.Analyzers/Conventional.Roslyn.Analyzers.csproj

```diff
@@ -5,8 +5,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="3.6.0" />
-      <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.6.0" />
+      <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="[4.0.1,)" />
```

Comment:
I'd prefer to use fixed versions for dependencies - it is important (to me) to be able to easily understand what versions of dependencies are in what build of the library. 

A solution I _should_ put in place for this is to set up dependabot and let it do its thing. I'll add that to my TODO list. For now, could you please change these versions to an appropriate fixed version?

Reply:
hopefully these particular versions are good enough